lesson6-questions.txt

```diff
@@ -42,6 +46,7 @@ capitalized? Why is the @ used?
 
 What is the difference between Forum.new() and Forum.create()?  Why can't you do Forum.save(...)?
 
+Forum.create saves a new forum and returns it instead of just returning a new forum. calling .save on the class itself doesn't have an instance to save.
```

Comment:
Yes! I'll just add a little extra detail that includes a few words/concepts that may be good to know or keep in mind. `Forum.create` returns a persisted (saved) instance of a `forum`, while `Forum.new` returns an instance that only exists in memory. And `save` is an instance method, there is no class method on `Forum` called `save`, so `Forum.save` would throw a `method_missing/undefined method` exception.